### ISSUE TITLE
Initiative roller only gives an error, dddice login non functional

### ISSUE THREAD

**Issue by Taldorin** [kamejosh/owlbear-hp-tracker#50]
Hitting any initiative roll  or any custom roll gives the error "Error while rolling dice - check if the selected dice theme is available" The dice theme i am using is still the free one "Bees" that came set up and is the only option I have. I attempted to use the login in the dice settings hoping that would fix it but upon getting sent to their activate site and putting the code inn i got a successful activation and then nothing at all, following its directions and going back changed nothing

**Comment by kamejosh** [comment]
@Taldorin 
Thx for reaching out, there are a couple of things I'd like to verify:

1. Do you have a dddice account
2. are you logged into that account with the browser you try to authenticate HP Tracker with
3. After entering the code does the popup in owlbear prompting the code automatically disapear?
4. On the top of the dice log does it still say "Login" or display your dddice account name: 
<img width="573" alt="image" src="https://github.com/user-attachments/assets/af8ae1d6-4a85-48fb-b9f7-1ecf2fee6db8">
5. do the custom dice button look something like this:
<img width="441" alt="image" src="https://github.com/user-attachments/assets/dc4dc2b9-6191-41d2-a514-8f8571730b8a">
and what is displayed while hovering abover one (see the 1d8 as an example in the screenshot)
6. Could you try to set the dice them to bees even though it's the one already selected just to make sure.
7. And last but not least could you open your browser console (F11 on chrome for windows) and screenshot the error message, maybe this gives me a better idea of what is going wrong because I cannot replicated the error.

**Comment by Taldorin** [comment]
Yes I created a dddice account and am logged into that account on the same browser. Upon entering the code that popup does go away after a few seconds but nothing changes it still says login after. Hovering over the dice produces the same result as the image you attached. I tried setting it to bees again but to no avail. Attached two images from the console note that I am using firefox, I also tested this on edge as well with the same results

![image](https://github.com/user-attachments/assets/3921e789-aa02-42fc-ba5f-e5e0ab8b25bb)


![image](https://github.com/user-attachments/assets/f3b2c3c3-c522-4696-a000-1a8bdec728f2)

**Comment by kamejosh** [comment]
@Taldorin 
Hmm interesting, so the error message is from dddice and says you want to roll a dice that is not available in the chosen theme. you could see the chosen theme  and the dice type in the `Request`  tab.

I think there might be some issue with the settings because the popup only auto closes once it gets the login token from dddice and at this point it should show you your dddice account name instead of "Login".

There is a way to verify this, you could open a private window and join the OBR room. This guest user should now automatically connect to dddice as a guest. So once the custom dice buttons appear you should be able to roll those buttons. This should work and if it does, I think the easiest solution to fix your problem might be, removing all current custom dice buttons. Next go to https://dddice.com/marketplace and add some of the free dice themes. now switch dice themes in HP Tracker and try again to roll initiative or something else.

My guess is that you had another theme set before dddice turned of access to community themes that are not in your dice box. HP Tracker now can't access this dice theme and tries to use bees (which is available for everybody) but something else fails then (I'm still trying to figure out what). But switching to another dice theme should at least fix the issue for you and I will try to fix it automatically once i figured out how to reproduce it.
